app/aws/mocks.py

```diff
@@ -55,7 +119,7 @@ def ses_notification_callback(reference):
         "MessageId": "8e83c020-1234-1234-1234-92a8ee9baa0a",
         "TopicArn": "arn:aws:sns:eu-west-1:12341234:ses_notifications",
         "Subject": None,
-        "Message": json.dumps(ses_message_body),
+        "Messages": [ses_message_body],
```

Comment:
we dont ned to convert this to json inside the messages?

Reply:
[I adjusted the lambda](https://github.com/cds-snc/notification-lambdas/pull/96/files#diff-f351401afceb63cdf76e273096f7e27431f5d72fc1dc889ae3ee479aa957b177R25) to take care of that for us instead.